pkg/grapicmd/di/app.go

```diff
@@ -88,3 +95,19 @@ func (c *appComponentImpl) Generator() module.Generator {
 	})
 	return c.generator
 }
+
+func (c *appComponentImpl) GexConfig() *gex.Config {
+	c.gexConfigOnce.Do(func() {
+		cfg := c.Config()
+		c.gexConfig = &gex.Config{
+			OutWriter:  cfg.OutWriter(),
+			ErrWriter:  cfg.ErrWriter(),
+			InReader:   cfg.InReader(),
+			FS:         cfg.Fs(),
```

Comment:
**[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 cannot use cfg.Fs() (value of type github.com/izumin5210/grapi/vendor/github.com/spf13/afero.Fs) as github.com/spf13/afero.Fs value in struct literal: wrong type for method Create